### PR TITLE
Add relayer functionality to fastauth

### PR DESCRIPTION
**Pull request by MaximusHaximus**
- Updates recovery service API calls to match advanced account API
- Implements near-social LAK-only functionality for fastauth accounts
- Adds relayer awareness to fastauth implementation
- Bumps NAJ to 2.1.3
- Bumps wallet-selector to latest versions

### CODE REVIEW

File: src/lib/selector/wallet.js [hunk 1]
```diff
@@ -1,118 +1,155 @@
...
+
+    const accountCreationData = JSON.parse(
+      window.localStorage.getItem("fast-auth:account-creation-data") ||
+        JSON.stringify({})
+    );
+
+    try {
+      if (
+        !accountCreationData.limitedAccessKey ||
+        !accountCreationData.isCreated
```
Comment (itegulov) [inline src/lib/selector/wallet.js:49]: I might be missing something here, but shouldn't this be:
```suggestion
        accountCreationData.isCreated
```

File: src/pages/AuthCallbackHandler.js [hunk 2]
```diff
@@ -1,111 +1,142 @@
...
+
+          const limitedAccessKey = nearAPI.KeyPair.fromRandom("ED25519");
+
+          const user = result.user;
+          if (!!user.emailVerified) {
+            setStatusMessage(
+              isRecovery ? "Recovering account..." : "Creating account..."
+            );
+
+            // TODO: Call MPC Service with accountId, publicKey,  and oauthToken to create account
```
Comment (itegulov) [inline src/pages/AuthCallbackHandler.js:41]: Isn't this TODO done?

File: src/pages/AuthCallbackHandler.js [hunk 3]
```diff
@@ -1,111 +1,142 @@
...
+
+            const options = {
+              method: "POST",
+              mode: "cors",
+              body: JSON.stringify(data),
+              headers,
+            };
+
+            await fetch(
+              `${MPC_RECOVERY_URL}/${isRecovery ? "add_key" : "new_account"}`,
```
Comment (itegulov) [inline src/pages/AuthCallbackHandler.js:71]: I don't think `add_key` supports `CreateAccountOptions`. Do you need us to add it?

**Review by itegulov — APPROVED** [review]

File: src/lib/selector/wallet.js (hunk 1 above)
Comment (MaximusHaximus) [reply src/lib/selector/wallet.js:49]: This case is a bit counter-intuitive -- after creating the account via recovery service, we redirect to the app after setting a flag indicating that the account was created successfully -- so this is correct, if the account wasn't created we can't yet fire the signIn() logic here

**Pull request merged**